pre_install_report/test/test_pre_install_report.py

```diff
@@ -39,13 +39,14 @@
 _RUNTIME_ERROR_RC_ = 7
 
 viya_kubelet_version_min = 'v1.14.0'
+viya_kubenetes_version = "1.22.0"
```

Comment:
This may be a nit.   I don't think there are actually .0 patch versions of k8s.   See https://kubernetes.io/releases/.   Every minor version seems to start with a .1 patch version.    I know, generally speaking you don't pay any attention to the patch version in your checks, but just wondering if this default value for viya_kubernetes_version might still be better set to a valid patch version.

Reply:
I think for the purposes of version comparison the .0 patch version is fine for conveying the purpose of the supported major.minor

Reply:
modified 1.22.0 to 1.22.4

Reply:
twas a nit comment ;)